Archives/ArchivePacker.h

```diff
@@ -29,5 +31,7 @@ namespace Archives
 		// Compares 2 filenames case insensitive to determine which comes first alphabetically.
 		// Does not compare the entire path, but only the filename.
 		static bool ComparePathFilenames(const std::string path1, const std::string path2);
+
+		void ArchivePacker::PackFile(StreamWriter& streamWriter, StreamReader& fileToPackReader, const uint64_t fileToPackSize);
```

Comment:
Prefix `ArchivePacker::` should be removed.

Reply:
I am terrible about this. Thanks.